README.md

```diff
@@ -233,18 +233,14 @@ Returns Placement for given parameters.
 
 ```javascript
 async () => {
-  const placement = await getPlacement(
-    "placement_id",
- 	 "LIVE",
- 	 "{ \"color\": \"blue\"}",
- 	 "campaign_id",
- 	 "experience_id"
-  );
- 
-  placement.impression();
-  ...
-  placement.clickthrough();
-  ...
+ const placement = await getPlacement(
+     "placement_id",
+ 	"LIVE",
+ 	"{ \"color\": \"blue\"}",
+ 	"campaign_id",
+ 	"experience_id"
+ );
+ ...
```

Comment:
Why calling impression() and clicktrhough() is removed from this example?

Reply:
This documentation is generated based on `index.ts` file on build. Added it there and (in consequence) here.